guides/common/Makefile

```diff
@@ -29,14 +29,17 @@ BROWSER_OPEN = open
 CP_ARGS =
 endif
 
-ifeq ($(LINKS), LOCAL)
 # The Satellite build uses links which do not resolve. This does mean the link
 # checker has issues with verification of guides that have not yet been built
 # by downstream, but there really isn't a way around this unless we start
 # building with downstream filenames.
 ifneq ($(BUILD), satellite)
+ifeq ($(LINKS), LOCAL)
 BASEURL_ATTR = -a "BaseURL=file://$(BUILD_DIRA)"
 endif
+ifneq ($(BASEURL), "")
+BASEURL_ATTR = -a "BaseURL=$(BASEURL)"
```

Comment:
doesn't this mean that production builds will also contain a localhost baseurl? I don't see anything in `.github/workflows/deploy.yml` that would unset `BASEURL` or set it to `docs.theforeman.org`?

Reply:
Uh, not sure I like abusing this implementation detail as a differentiator.

Maybe document the var in the readme and explicitly set it to "" in the deployment pipeline?

Reply:
@evgeni @ekohl Hiii! Is Ewoud still available to help you finish this? Or should we try to ask someone else? :)

Reply:
@evgeni how about this: you do need to pass an env var to opt-in, but at least the behavior is consistent

Reply:
sounds good to me